lib/app/features/chat/messages/views/components/message_types/money_message/components/amount_display.dart

```diff
@@ -41,11 +41,7 @@ class _AmountDisplay extends HookWidget {
             ),
             SizedBox(width: 4.0.s),
             Text(
-              '~ ${NumberFormat.currency(
-                locale: 'en_US',
-                symbol: '',
-                decimalDigits: 2,
-              ).format(equivalentUsd)} USD',
+              '~ ${formatUSD(equivalentUsd)} USD',
```

Comment:
Are we tied to `USD`?

Reply:
I think yes, but specifically here I just added a universal method for formatting doubles into the USD currency format, nothing more.